Assets/PatchKit Patcher/Scripts/App.cs

```diff
@@ -163,7 +163,7 @@ private static IDownloadDirectory CreateDefaultDownloadDirectory(string appDataP
 
         private static ILocalMetaData CreateDefaultLocalMetaData(string appDataPath)
         {
-            return new LocalMetaData(appDataPath.PathCombine("patcher_cache.json"));
+            return new LocalMetaData(appDataPath.PathCombine("patcher_data.json"), appDataPath.PathCombine("patcher_cache.json"));
```

Comment:
Now I'm thinking whether shouldn't we call it `app_data.json`

What do you think @sp0lsh @genail ?

Reply:
@sp0lsh @genail bump

Reply:
Sorry @witcher112. Github notifications are one of the worst to notice...
Well, app_data.json sounds actually quite good. This file is for the app to access it, so only application-related data will be there.

Reply:
OK 👍 
@sp0lsh can you make such modification?